articles/custom_storage_location.md

```diff
@@ -42,7 +40,7 @@ To allow authorized Synapse users to upload data to your bucket set read-write p
 {
     "Statement": [
         {
-            "Action": "s3:ListBucket*",
+            "Action": [ "s3:ListBucket*", "s3:GetBucketLocation" ],
```

Comment:
Can you confirm that these changes won't affect existing buckets in the currently allowed region, `us-east-1`?